src/components/react-admin/DraggableFormIterator.js

```diff
@@ -205,9 +213,7 @@ export class DraggableFormIterator extends Component {
     // the fields prop which will always be empty for a new record.
     // Without it, our ids wouldn't match the default value and we would get key warnings
     // on the CssTransition element inside our render method
-    this.ids = this.nextId > 0
-      ? Array.from(Array(this.nextId).keys())
-      : [];
+    this.ids = this.nextId > 0 ? Array.from(Array(this.nextId).keys()) : [];
```

Comment:
For the same goal, maybe this is more readable to understand the meaning:
`this.ids = this.nextId > 0 ? Array.from({ length: this.nextId }, (_, index) => index) : [];`

Reply:
I think I can also remove the "this.nextId>0" as it's safe to call this with `0` value.

I propose some shorter alternatives. Which one do you prefer ?

```js
// 1 Use spreading operator
this.ids = [...Array(5).keys()];

// 2 Use lodash range
import range from 'lodash'
this.ids = range(5);

Reply:
I also think it's safe but i don't really know how ids are defined so i though it's not bad to keep `> 0`. Moreover, it gives the information that `this.nextId` is an integer type.

About 1/, i prefer my request, but it's equal.
About 2/, as we said IRL, the use of lodash is a little overkill.